README.md

```diff
@@ -132,3 +132,5 @@ Please read [CONTRIBUTING.md](#CONTRIBUTING.md) before you help us!
 [ jmakhack ](https://github.com/jmakhack)
 
 [ odysseaspapadimas ](https://github.com/odysseaspapadimas)
+
```

Comment:
I previously mentioned to send this file in a separate PR. Not part of this issue

Reply:
@Jasonwei1106 remove this file from checking in